source/gui/_addonStoreGui/viewModels/store.py

```diff
@@ -241,6 +241,21 @@ def _makeActionsList(self):
 				validCheck=lambda aVM: isinstance(aVM.model, _AddonStoreModel),
 				actionTarget=selectedListItem
 			),
+			AddonActionVM(
+				# Translators: Label for an action that opens the webpage to see and send feedback for the selected add-on
+				displayName=pgettext("addonStore", "Read or provide &feedback"),

```

Comment:
Please add an appropriate accelerator key, but this should just be "Reviews"
```suggestion
				displayName=pgettext("addonStore", "Reviews"),
```